packages/gatsby-source-wordpress/README.md

```diff
@@ -86,6 +86,14 @@ The Authentication on Wordpress.com is not supported yet. This means that you wo
           user: 'your-htaccess-username',
           pass: 'your-htaccess-password',
           sendImmediately: false
+
+          // If hostingWPCOM is true then you will need 
+          // to communicate with wordpress.com api
+          // in order to do that you need to create an app
+          // at https://developer.wordpress.com/apps/
+          // then add the clientId and clientSecret here
+          clientId: '',
```

Comment:
Inside auth makes sense. Maybe we should have all the options prefixed so it's still clear for everyone in 3 month.
```
auth: {
          htaccess_user: 'your-htaccess-username',
          htaccess_pass: 'your-htaccess-password',
          htaccess_sendImmediately: false,

          // If hostingWPCOM is true then you will need 
          // to communicate with wordpress.com api
          // in order to do that you need to create an app
          // at https://developer.wordpress.com/apps/
          // then add the clientId and clientSecret here
          wpcom_app_clientId: '',
          wpcom_app_clientSecret: ''
        },```